index.html

```diff
@@ -193,6 +194,39 @@ <h2>ts-note</h2>
 				</script>
 			</article>
 
+			<article>
+				<h2>ts-tabs</h2>
+				<ts-tabs dir="rtl" id="tabs1"></ts-tabs>
+				<script>
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') The length of this JS script (27) exceeds the maximum set to 5. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=Web%3ALongJavaScriptCheck)